docs/sphinx/examples/python_runtime/noise_amplitude_damping.py

```diff
@@ -22,7 +22,7 @@
 
 
 # Now we may define our simple kernel function and allocate a qubit.
-@cudaq.kernel(jit=True)
+@cudaq.kernel(jit=False)
```

Comment:
Why go from true to false on this one?

Reply:
This was a typo, all the examples under `python_mlir` had `jit=True` and those under `python_runtime` either relied on default (which was eager mode) or `jit=False`.